bitnami/victoriametrics/values.yaml

```diff
@@ -0,0 +1,3662 @@
+      ##
+      adaptSecurityContext: auto
+## @section Common parameters
+##
+
+## @param kubeVersion Override Kubernetes version
+##
+kubeVersion: ""
+## @param nameOverride String to partially override common.names.name
+##
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+##
+fullnameOverride: ""
+## @param namespaceOverride String to fully override common.names.namespace
+##
+namespaceOverride: ""
+## @param commonLabels Labels to add to all deployed objects
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+## @param extraDeploy Array of extra objects to deploy with the release
+##
+extraDeploy: []
+
+## @section VictoriaMetrics common parameters
+##
+## @param envflagEnable Enable envflag
+##
+envflagEnable: true
+## @param envflagPrefix Prefix used for the envflag entries
+##
+envflagPrefix: VM_
+## @param loggerFormat Set format of the logs
+##
+loggerFormat: json
+
+## @section VictoriaMetrics Select Parameters
+##
+vmselect:
+  ## Bitnami VictoriaMetrics Select image
+  ## ref: https://hub.docker.com/r/bitnami/victoriametrics-vmselect/tags/
+  ## @param vmselect.image.registry [default: REGISTRY_NAME] VictoriaMetrics Select image registry
+  ## @param vmselect.image.repository [default: REPOSITORY_NAME/victoriametrics-vmselect] VictoriaMetrics Select image repository
+  ## @skip vmselect.image.tag VictoriaMetrics Select image tag (immutable tags are recommended)
+  ## @param vmselect.image.digest VictoriaMetrics Select image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag image tag (immutable tags are recommended)
+  ## @param vmselect.image.pullPolicy VictoriaMetrics Select image pull policy
+  ## @param vmselect.image.pullSecrets VictoriaMetrics Select image pull secrets
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/victoriametrics-vmselect
+    tag: 1.113.0-debian-12-r0
+    digest: ""
+    ## Specify a imagePullPolicy
+    ## ref: http://kubernetes.io/docs/concepts/containers/images/#pre-pulled-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+  ## @param vmselect.replicaCount Number of VictoriaMetrics Select replicas to deploy
+  ##
+  replicaCount: 1
+  ## @param vmselect.containerPorts.http VictoriaMetrics Select http container port
+  ##
+  containerPorts:
+    http: 8481
+  ## Network Policies
+  ## Ref: https://kubernetes.io/docs/concepts/services-networking/network-policies/
+  ##
```

Comment:
We usually include the Network Policies block under "Traffic Exposure Parameters" section.